src/client/localplayer.cpp

```diff
@@ -271,7 +257,8 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 		is_climbing = false;
 	} else {
 		is_climbing = (nodemgr->get(node.getContent()).climbable ||
-			nodemgr->get(node2.getContent()).climbable) && !free_move;
+							  nodemgr->get(node2.getContent()).climbable) &&
```

Comment:
This line doesn't seem to be aligned with anything, but is indented by 5 tabs and a couple of spaces.